builtin/providers/aws/resource_aws_load_balancer_policy.go

```diff
@@ -0,0 +1,240 @@
+package aws
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/hashicorp/terraform/helper/hashcode"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsLoadBalancerPolicy() *schema.Resource {
+	return &schema.Resource{
+		// There is no concept of "updating" a Load Balancer Policy in
+		// the AWS API.
+		Create: resourceAwsLoadBalancerPolicyCreate,
+		Update: resourceAwsLoadBalancerPolicyCreate,
```

Comment:
As of 6e4b445b5852ac75081063ca553a148492cf3944 have a test that requires that there be no ``Update`` callback when all of the attributes are ``ForceNew``, so if you rebase your change to master I think you'll find that this won't pass tests anymore.

The easy fix is to just drop this line. ``Update`` can never be called if all attributes ``ForceNew``, so it's fine (and now required!) to omit it.